src/extension/background-script/actions/lnurl/index.ts

```diff
@@ -22,14 +22,9 @@ async function lnurl(message: Message | MessageLNURLChannel) {
 
   switch (lnurlDetails.tag) {
     case "channelRequest":
-      if ("public" in message) {
-        return channelRequestWithPrompt(message, lnurlDetails);
-      } else {
-        console.error("Wrong 'message' type: ", message);
-        return;
-      }
+      return channelRequestWithPrompt(message, lnurlDetails);
     case "login":
-      return authWithPrompt(message, lnurlDetails);
+      return authWithPrompt(message, lnurlDetails); // authOrPrompt
```

Comment:
is this still a todo, or is this fine?

Reply:
Nope, thanks to the reminder. Renamed this.